app/views/species_list/download.html.erb

```diff
@@ -35,8 +36,8 @@
   <%= submit_tag(:species_list_report_button.l, class: "btn") %>
 </form>
 
-<%= form_tag(add_query_param({ controller: "observer",
-                               action: "download_observations" }, @query)) %>
+<%= form_tag(add_query_param({ controller: :observations,
+                               action: :download }, @query)) %>
```

Comment:
If I had a dollar for every time we've switched back and forth between strings and symbols on these!!